geniza/pages/admin.py

```diff
@@ -12,4 +13,10 @@
 
 admin.site.unregister(Image)
 admin.site.unregister(Document)
-admin.site.register(Contributor)
+
+# Set up tabbed translation admin for contributors
+@admin.register(Contributor)
+class ContributorAdmin(TabbedTranslationAdmin):
+    list_display = ("last_name", "first_name", "role")
+    search_fields = ("first_name", "last_name", "role")
+    fields = ("last_name", "first_name", "role")
```

Comment:
I wonder if this should be made editable in wagtail admin instead of django admin, but IDK if wagtail admin supports model translation! So let's leave it here for now.

Reply:
Yeah, I did try this originally, but the Wagtail admin doesn't play nicely with non-Wagtail models. And it didn't seem to make sense to have Contributor as a subclass of Page or Document… not sure!

Reply:
There's a way to register models that aren't pages to be edited in wagtail — Nick set it up in our CDH website code base. We can look at that at some point and see whether the approach works with translatable models, but I think this is workable for now.

Reply:
Oh, great! Yes, that would be better in the future so that content editors don't have to go to multiple places.